resolwe/flow/tests/test_models.py

```diff
@@ -766,6 +800,10 @@ def test_hydrate_input_references(self):
             },
             size=0,
         )
+        data_location = DataLocation.objects.create(subpath='')
```

Comment:
You should use a non-empty path for testing that path is rendered correctly.

Reply:
The path gets set one line below.